docs/source/model/manual_optimization.rst

```diff
@@ -186,8 +186,8 @@ Use ``self.lr_schedulers()`` in  your :class:`~pytorch_lightning.core.module.Lig
 defined in your :meth:`~pytorch_lightning.core.module.LightningModule.configure_optimizers`.
 
 .. warning::
-   * Before v1.3, Lightning automatically called ``lr_scheduler.step()`` in both automatic and manual optimization. From
-     1.3, ``lr_scheduler.step()`` is now for the user to call at arbitrary intervals.
+   * Before v1.3, Lightning automatically called ``lr_scheduler.step()`` in both automatic and manual optimization. 1.3, ``lr_scheduler.step()`` can now be called at arbitrary intervals by the user, or automatically by Lightning if ``"interval"`` is defined in ``configure_optimizers()``.
```

Comment:
This is not correct during manual optimization, we actually show a warning:

```python
/home/carmocca/git/py39/lib/python3.9/site-packages/pytorch_lightning/core/optimizer.py:327: RuntimeWarning: The lr scheduler dict contains the key(s) ['interval'], but the keys will be ignored. You need to call `lr_scheduler.step()` manually in manual optimization.
```

Reply:
Hi @lukasugar, the section pointed out by carmocca is about manual optimization, so the following statement is always false in manual optimization. (while it's true in automatic optimization.)

> automatically by Lightning if ``"interval"`` is defined in ``configure_optimizers()``.